package.xml

```diff
@@ -2,7 +2,7 @@
 <package format="2">
   <name>robot_localization</name>
   <version>2.3.1</version>
-  <description>The robot_localization package provides nonlinear state estimation through sensor fusion of an abritrary number of sensors.</description>
+  <description>Nonlinear state estimation through sensor fusion of an abritrary number of sensors.</description>
```

Comment:
It won't. However, when the linter complains, you can make note of the issue category (usually given in brackets at the end of the error when it is reported) and [filter out](http://wiki.ros.org/roslint#Tips) that warning in the CMakeLists.txt.